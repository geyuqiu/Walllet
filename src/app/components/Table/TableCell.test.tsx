import {render} from "@testing-library/react";
import React from "react";

import {TableCell} from './TableCell';

describe("TableRow", () => {
	it("should render with border", () => {
		const { container } = render(
			<table>
				<tbody>
					<tr>
						<TableCell>1</TableCell>
					</tr>
				</tbody>
			</table>,
		);
		expect(container).toMatchSnapshot();
	});
});
