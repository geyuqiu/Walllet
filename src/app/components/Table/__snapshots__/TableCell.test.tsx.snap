// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TableRow should render with border 1`] = `
<div>
  <table>
    <tbody>
      <tr>
        <td>
          <div
            class="TableCell__TableCellInnerWrapper-sc-12y9mmp-0 dbZfon"
          >
            1
          </div>
        </td>
      </tr>
    </tbody>
  </table>
</div>
`;
